projects/components/src/time-range/time-range.component.test.ts

```diff
@@ -150,4 +150,17 @@ describe('Time range component', () => {
     expect(refreshButton.label).toBe('Refresh - updated 6m ago');
     discardPeriodicTasks();
   }));
+
+  test('should set seconds and milliseconds to zero', () => {
```

Comment:
Can we also add a test case for relative time range too? That will ensure that both `sec` and `ms` are zero.

Reply:
✅ Added test for relative time range